lib/full-site-editing/template-parts.php

```diff
@@ -189,6 +192,16 @@ function gutenberg_get_allowed_template_part_areas() {
 			'icon'        => 'header',
 			'area_tag'    => 'header',
 		),
+		array(
+			'area'        => WP_TEMPLATE_PART_AREA_PRIMARY_MENU,
```

Comment:
There could be an area per semantic menu type, or it could be more of a new, free field that can take any value..